crates/state/src/write_log.rs

```diff
@@ -222,7 +172,7 @@ impl WriteLog {
     pub fn read_persistent(
```

Comment:
Should this me named just `read`? Just to match the naming scheme of the write methods which are `write` and `write_temp`

Reply:
good call, will do!

Reply:
done in [791d172](https://github.com/anoma/namada/commit/791d17282c294b6cb7d0b0be1caf318df4886bcb) - also removed an unused err case

Reply:
[172ba75](https://github.com/anoma/namada/commit/172ba7553dc472044d4933116e44da06220b14b5)